### PR TITLE
#761 rename tag to token

### PR DESCRIPTION
**Pull request by saadmhybris**
Closes sap/fundamental#761

Renamed `fd-tag` to `fd-token` 

#### Test

* http://localhost:3030/token

#### Changelog

**New**

nothing 

**Changed**

* tag to token

**Removed**

nothing

### CODE REVIEW

**Comment by joseegm** [comment]
Test Version for this PR was deployed

Built with commit b5a22e0427cba99df3299030a455af07cb11aa6e

https://deploy-preview-803--fundamental.netlify.com

File: docs/pages/components/tag.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ permalink: components/tag.html
 folder: components
 summary:
 ---
+<span class="fd-badge fd-badge--error fd-badge--pill">Depricated</span>
 
-Tags are used to represent contextualizing information. They can be useful to show applied filters, selected values for form field or object metadata.
-
+This componnet has been renamed to <a href="token.html">Token</a>
```
Comment (xak) [inline docs/pages/components/tag.md]: `component`

File: scss/components/token.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+///////////////////////////////////////////////////////////////////////////////
+
+/*!
+.fd-tag+(--no-border)
+    .fd-tag__content+()
+    .fd-tag__title+()
+*/
+$block: #{$fd-namespace}-token;
+
+.#{$block} {
```
Comment (xak) [inline scss/components/token.scss]: Just remove the `tag` files and just add the class here.

```suggestion
$block: #{$fd-namespace}-token;
$block-alt: #{$fd-namespace}-tag;

.#{$block},
.#{$block-alt} {
```

There's no need to output 2x the CSS.

**Comment by saadmhybris** [comment]
@xak ready for review again.

**Pull request merged**